### PR TITLE
Monitoring / Maintenance / Reports

### PR DESCRIPTION
**Pull request by erasche**
- split monitoring + maintenance slides into monitoring, maintenance, other
- add reports tutorial
- add install for gxadmin
- add telegraf tuto

### CODE REVIEW

File: topics/admin/tutorials/stuff/slides.html [hunk 1]
```diff
@@ -1,29 +1,3 @@
...
-contributors:
-  - natefoo
-  - bgruening
-  - slugger70
----
-layout: true
-name: left-aligned
-class: left, middle
----
-
```
Comment (nsoranzo) [inline topics/admin/tutorials/stuff/slides.html]: This is missing metadata (not on purpose, I suppose).

File: topics/admin/tutorials/maintenance/slides.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Galaxy-specific check in `contrib/nagios/`: Runs Galaxy jobs
+
+---
+
+# Sentry
+
+* Motto: *"Stop hoping your users will report errors"*
+* Error tracking and analysing tool.
+* Galaxy has Sentry middleware that you can enable in configuration.
```
Comment (nsoranzo) [inline topics/admin/tutorials/maintenance/slides.html]: I think Nagios and probably also Sentry should be moved to the monitoring slide deck, but I'm not using either yet, so feel free to ignore.

File: topics/admin/tutorials/stuff/slides.html [hunk 3]
```diff
@@ -337,6 +181,13 @@
 Galaxy-specific check in `contrib/nagios/`: Runs Galaxy jobs
 
 ---
+# Sentry
+
+* Motto: *"Stop hoping your users will report errors"*
+* Error tracking and analysing tool.
+* Galaxy has Sentry middleware that you can enable in configuration.
+---
```
Comment (nsoranzo) [inline topics/admin/tutorials/stuff/slides.html]: This slide is duplicated in another slide deck, remove.

File: topics/admin/tutorials/stuff/slides.html (hunk 1 above)
Comment (erasche) [reply topics/admin/tutorials/stuff/slides.html]: Somewhat on purpose, hence the name `stuff` as well, it was just incoherent random stuff extracted from monitoring/maintenance that weren't directly related. Most of it seems to fall under "admin of galaxy tips" but I don't know what to do with them, so stuck them in a 'stuff' folder under someone needed them and had time to make them into a coherent story. I'll re-add the metadata with `enabled: false`, but if you have any good ideas I'd love to hear them!

**Pull request merged**